javascript/package.json

```diff
@@ -25,6 +26,9 @@
     "typescript": "4.4.3"
   },
   "dependencies": {
-    "@gomomento/sdk": "0.13.0"
+    "@gomomento/generated-types": "0.20.0",
+    "@gomomento/sdk": "file:../../client-sdk-javascript",
```

Comment:
^ I assume this was temporary since you were changing the SDK in tandem. :reminder_ribbon: to revise later.

Reply:
yup, build won't pass until i fix this, but was planning on fixing it after we do a release of the actual SDK with the tuning fixes.

Reply:
updated to released version of JS SDK in d423998132f04e68fb33060da3821826b95fc37c